modules/bim/lib/open_project/bim/engine.rb

```diff
@@ -214,10 +216,9 @@ class Engine < ::Rails::Engine
     end
 
     config.to_prepare do
-      Doorkeeper.configuration.scopes.add(:bcf_v2_1)
-
       unless defined? OpenProject::Authentication::Scope::BCF_V2_1
         OpenProject::Authentication::Scope::BCF_V2_1 = :bcf_v2_1
+        Doorkeeper.configuration.scopes.add(OpenProject::Authentication::Scope::BCF_V2_1)
```

Comment:
:yellow_circle: In which context did this change occur? I can't see a separate commit for it and I am wondering whether this is intended as a cleanup or whether something didn't work as intended.

I would have expected that adding the scope to the doorkeeper configuration was required on each reload, therefore it was included in the front of `config.to_prepare`.

On the other hand constants remain defined even when rails reloads its config, so I'd expect that the new code now effectively only adds the scopes once to doorkeeper, not on each reload.

Is this intentional?